src/main/features/claimant-response/routes/confirmation.ts

```diff
@@ -17,6 +69,8 @@ export default express.Router()
         {
           claim: claim,
           confirmationDate: MomentFactory.currentDate(),
-          repaymentPlanOrigin: getRepaymentPlanOrigin(claim.settlement)
+          repaymentPlanOrigin: claim.settlement && getRepaymentPlanOrigin(claim.settlement),
+          countyCourtJudgement: claim.countyCourtJudgmentIssuedAt !== undefined,
```

Comment:
Please remove and use `claim.countyCourtJudgmentIssuedAt` in NJK templates. A `countyCourtJudgement` is just misleading.